tests/test_cluster.py

```diff
@@ -185,7 +188,7 @@ def test_fanout_api(cluster):
 def test_fanout_key_target(cluster):
     with cluster.fanout() as client:
         c = client.target_key('foo')
-        c.set('foo', str(42))
+        c.set('foo', bytes_type(42))
```

Comment:
Are you sure they want bytes here?  When using `set` the values will be string. It looks to me like they want `42` to be a string like `"42"`.

Reply:
I copied this over from dcramers stuff fwiw

Reply:
In Py2:

```
In [9]: str(42)
Out[9]: '42'
```

In Py3
```
>>> bytes(42)
b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'
```

Reply:
lol you're right